src/NuGet.Clients/PackageManagement.VisualStudio/VSSettings.cs

```diff
@@ -13,7 +13,23 @@ namespace NuGet.PackageManagement.VisualStudio
     [PartCreationPolicy(CreationPolicy.Shared)]
     public class VSSettings : Configuration.ISettings
     {
-        private Configuration.ISettings SolutionSettings { get; set; }
+        // to initialize SolutionSettings first time outside MEF constructor
+        private Configuration.ISettings _solutionSettings;
+
+        private Configuration.ISettings SolutionSettings
```

Comment:
This class isn't thread-safe anyway. I see race conditions orthogonal to this change. I hope no one calls into an initialized copy of it off the UI thread. At least not without knowing exactly what is thread-safe and what isn't.
I can elaborate on the shortcomings of it if you need.